core_lib/src/structure/layerbitmap.cpp

```diff
@@ -44,6 +44,11 @@ BitmapImage* LayerBitmap::getLastBitmapImageAtFrame(int frameNumber, int increme
     return static_cast<BitmapImage*>(getLastKeyFrameAtPosition(frameNumber + increment));
 }
 
+void LayerBitmap::replaceKeyFrame(const KeyFrame* bitmapImage)
+{
+    *getLastBitmapImageAtFrame(bitmapImage->pos()) = *static_cast<const BitmapImage*>(bitmapImage);
```

Comment:
Can we use `getBitmapImageAtFrame()` instead of `getLastBitmapImageAtFrame()`? I think the bitmapImage should be just at the specified position for replacement.

Reply:
Good point, yes we might be able to do that